packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -226,8 +227,11 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     wire.setKeepAlive(true); //  enables the keep-alive ping (triggered every 60s).
     wire.setTimeout(65000); // sets the requests timeout to 65seconds
     wire.on(WireEvents.KEEP_ALIVE, () => {
-      log.debug({keepAlive: !torrent.done && wire.amChoking}, 'wire keep-alive');
-      log.trace({torrent});
+      {
+        const msg = 'wire keep-alive';
+        log.debug({keepAlive: !torrent.done && wire.amChoking}, msg);
+        log.trace({torrent, wire}, msg);
```

Comment:
Not hugely important, but does this mean that the wire is double logged? The torrent contains wires, correct?

Reply:
Yes. Maybe it's useful to know it's _this_ wire that we're keeping alive.


Maybe it's not useful at all to log torrents. I don't know 👀